webpay/auth/views.py

```diff
@@ -191,4 +191,10 @@ def fxa_login(request):
         request.POST.get('auth_response'))
     log.info("FxA login response:" + repr(data,))
     user_hash = set_user(request, data['email'], verified=True)
-    return {'user_hash': user_hash, 'user_email': data['email']}
+
+    super_powers = data['email'] in settings.USERS_WITH_SUPER_POWERS
+    log.info('user has super powers? {user}: {powers}'
+             .format(user=data['email'], powers=super_powers))
+
+    return {'user_hash': user_hash, 'user_email': data['email'],
+            'super_powers': super_powers}
```

Comment:
Yeah. This is just a WiP so I could show Stuart since I was stuck on the spartacus side. I'll fix this up once it's working.